packages/dapp-high-roller/src/components/app-wager/app-wager.tsx

```diff
@@ -249,11 +260,12 @@ export class AppWager {
               class="form__input"
               id="betAmount"
               type="number"
-              placeholder="3 ETH"
+              placeholder="0.01"
               value={this.betAmount}
               onInput={e => this.handleChange(e, "betAmount")}
-              readonly={true}
-              step="0.01"
+              min={0}
+              max={0.01}
+              step={0.001}
```

Comment:
So the value can't be something like `0.00001` if step is `0.001`?

Reply:
It can, but you have to type it manually.